snap/snapcraft.yaml

```diff
@@ -329,8 +341,9 @@ apps:
     adapter: full
     command: bin/app-service-configurable -confdir $SNAP_DATA/config/app-service-configurable/res -profile rules-engine --registry
     daemon: simple
+    environment:
+      Binding_PublishTopic: events
```

Comment:
FYI, I started to have concern that this would break the ability to switch to old rules engine that subscribes to blank topic, but I think the blank topic is like a wild card topic for ZMQ. Thoughts?

Reply:
I just confirmed this is correct:
```
A consequence of this prefix matching behaviour is that you can receive all published messages by subscribing with an empty topic string.
```

Reply:
OK, thanks!